assets/css/main.css

```diff
@@ -2531,11 +2543,11 @@ input, select, textarea {
 
 	#header {
 		text-align: center;
-		padding: 3em 0 0 0;
-		background-color: #fff;
-		background-image: url("images/bg02.png"), url("images/bg02.png"), url("images/bg01.png");
-		background-position: top left,					top left,					top left;
-		background-size: 100% 6em,					100% 6em,					auto;
+		padding: 4em 0 0 0;
+		background-color: var(--headerbg);
+		background-image: url("../../images/banner2.jpg");
```

Comment:
Why did this path change?

Reply:
The previous path pointed towards 3 transparent "texture" images and then would just take on the background color, to create a textured background of that colour. Before change:
![image](https://user-images.githubusercontent.com/85590111/121455936-a7e69900-c97b-11eb-8e46-bb82d8fee21c.png)
The new path points towards banner2.jpg which was a new file created for the banner, that contains the logo. It was updated on main.css so it would apply to each page that contains the banner. After change:
![image](https://user-images.githubusercontent.com/85590111/121456196-1297d480-c97c-11eb-8167-90f6c42b8f82.png)

Reply:
Would it work without the ../.. relative pathing stuff?

Reply:
Another banner image was in use elsewhere on the site. Banner2 was added to the same folder, and matched the pathing used for it. Presumably it is required as the images aren't stored in a subfolder below where the main.css is stored, so it needs to path "up" folders with the ../.., however I did not see personally check if it is required, I just implemented it in the same way as the other was.
![image](https://user-images.githubusercontent.com/85590111/121457203-acac4c80-c97d-11eb-8417-160087587df4.png)

Reply:
👍 
As long as it is intentional and doesn't break things I'm cool with it